cmd/ci-secret-bootstrap/main.go

```diff
@@ -137,11 +137,24 @@ func (o *options) validateCompletedOptions() error {
 			if bwContext.BWItem == "" {
 				return fmt.Errorf("config[%d].from[%s]: empty value is not allowed", i, key)
 			}
-			if bwContext.Field == "" && bwContext.Attachment == "" {
-				return fmt.Errorf("config[%d].from[%s]: either field or attachment needs to be non-empty", i, key)
+			if bwContext.Attribute != "" && bwContext.Attribute != "password" {
+				return fmt.Errorf("config[%d].from[%s]: not supported attribute ('password'): %s", i, key, bwContext.Attribute)
```

Comment:
Maybe something like `"config[%d].from[%s].attribute: only the 'password' is supported, not %s"`

Reply:
Done.